app/enterprise/1.3-x/developer-portal/theme-customization/single-page-app.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Hosting Single Page App out of Kong Dev Portal
+---
+
+### Introduction
+
+The Kong Developer Portal ships with a default server side rendered theme, however it is possible to replace this with a Single Page App(SPA). Our example is in Angular using the Angular CLI Tool, but you can follow along with any other javascript framework, with just a few tweaks.
+
+To view the basic example Angular template from this this guide view the [`example/spa-angular` branch from the kong-portal-templates branch](https://github.com/Kong/kong-portal-templates/tree/example/spa-angular).
+
+### Prerequisites
+
+* Kong Enterprise 1.3 or later
+* Portal Legacy is turned off
+* The Developer Portal is enabled and running
+* kong-portal-cli tool is installed locally
+
+### What is a SPA
+
+A Single Page App(SPA) is a website that loads all the HTML, Javascript, and CSS on the first load. Instead of loading subsequent pages from the server, javascript is used to dynamically change the page content. You may wish to use a SPA in Dev Portal if you have a preexisting SPA you want to integrate with portal, or you are trying to achieve a more application like experience across many pages. A SPA takes control of routing from the server, and handles it client side instead.
```

Comment:
```suggestion
A Single Page App (SPA) is a website that loads all HTML, Javascript, and CSS on the first load. Instead of loading subsequent pages from the server, javascript is used to dynamically change the page content. You may wish to use a SPA in Dev Portal if you have a preexisting SPA you want to integrate with portal, or you are trying to achieve a more application like experience across many pages. A SPA takes control of routing from the server, and handles it client-side instead.
```